### PR TITLE
Include user specified themes

### PR DESCRIPTION
**Pull request by danBamikiya**
<h4>I added a feature that allows users specify which themes chameleon should switch through (VS Code has a lot of default 'ugly' themes so users may want to specify which themes chameleon can switch through rather than all available themes).</h4>

---

I also added some important tests for the extension and separated build task from test task (in tasks.json and launch.json).

### CODE REVIEW

**Review by timdeschryver — COMMENTED** [review]
Hi! Thanks for your contribution, and making some needed changes to make this project "contribution ready".

I was thinking about adding more options, one of which being to filter themes.
Before we merge this PR, I want to ask you if it would make more sense to add a exclude option, rather than an include option. The reasoning behind this, is I think that it's harder to maintain an include list. Every time you install a theme, you would have to think about adding the new theme to this list. But then I was also thinking if it would make more sense to add a `includeBuiltInThemes` option, to enable/disable the built in VSCode themes.

**Comment by danBamikiya** [comment]
Hi! I think the exclude option is smarter(I didn't think about that 😄) and the includeBuiltInThemes option would be perfect!

I'm also currently working on a switchThemesInterval option that allows users specify what intervals(hours) to switch to the next theme... I'm currently running tests for the feature!

**Comment by danBamikiya** [comment]
I'll change the feature to an excludeUserDefinedThemes

**Comment by danBamikiya** [comment]
I've changed it. Users can now exclude specific themes

**Review by timdeschryver — APPROVED** [review]

**Pull request merged**
